data-reporting-service/post-processing-service/src/test/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingServiceTest.java

```diff
@@ -59,7 +61,7 @@ public void testPostProcessPatientMessage() {
         postProcessingServiceMock.postProcessPatientMessage(key, topic);
         postProcessingServiceMock.processCachedIds();
 
-        String expectedPatientIdsString = "123"; //String.join(",", patientIds.stream().map(String::valueOf).collect(Collectors.toList()));
+        String expectedPatientIdsString = "123";
         verify(patientRepositoryMock).executeStoredProcForPatientIds(expectedPatientIdsString);
         verify(patientRepositoryMock, times(1)).executeStoredProcForPatientIds("123");
```

Comment:
you can use `expectedPatientIdsString` here instea of hardcoding.

Reply:
will fix it